freedesktop/com.github.blackhole89.notekit.metainfo.xml

```diff
@@ -0,0 +1,76 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+	<id>com.github.blackhole89.notekit.desktop</id>
+	<metadata_license>MIT</metadata_license>
+	<project_license>MIT</project_license>
```

Comment:
Project license is GPLv3, though (and probably, the metadata should be by default, too...?

Reply:
The metadata should be released under a license that would allow almost every distributor to display those information on their store, for more details see https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html#tag-metadata_license. Typically metainfo files uses CC0-1.0

Reply:
Ah, I see. I don't mind the metainfo being CC0-1.0. The files, I guess, are GPL-3.0-or-later (I probably should add the standard blurb to all of them, since I realised the GPLv3 LICENSE file does not actually specify if it's "only" or "or later"!).